### PR TITLE
[Notifier] Add new Pushy notifier bridge

### PR DESCRIPTION
**Pull request by stloyd**
| Q             | A
| ------------- | ---
| Branch?       | 7.1
| Bug fix?      | no
| New feature?  | yes 
| Deprecations? | no 
| License       | MIT

This PR adds a new bridge for [Pushy](https://pushy.me). Based on documentation available: https://pushy.me/docs/api/send-notifications

### CODE REVIEW

File: src/Symfony/Component/Notifier/Bridge/Pushy/PushyOptions.php [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+/**
+ * @author Joseph Bielawski <stloyd@gmail.com>
+ *
+ * @see https://pushy.me/docs/api/send-notifications
+ */
+final class PushyOptions implements MessageOptionsInterface
+{
+    private const INTERRUPTION_LEVEL = ['passive', 'active', 'time-sensitive', 'critical'];
+
+    public function __construct(private array $options = [])
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/Pushy/PushyOptions.php]:
```suggestion
    public function __construct(
        private array $options = [],
    )
```

File: src/Symfony/Component/Notifier/Bridge/Pushy/Tests/PushyTransportTest.php [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        yield [new ChatMessage('Hello!')];
+        yield [new DummyMessage()];
+    }
+
+    public function testSendWithOptions()
+    {
+        $messageSubject = 'testMessageSubject';
+        $messageContent = 'testMessageContent';
+
+        $response = $this->createMock(ResponseInterface::class);
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/Pushy/Tests/PushyTransportTest.php]: Can you please use `JsonMockResponse`?

File: src/Symfony/Component/Notifier/Bridge/Pushy/Tests/PushyTransportTest.php [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+        $this->assertSame('123', $sentMessage->getMessageId());
+    }
+
+    public function testSendWithNotification()
+    {
+        $messageSubject = 'testMessageSubject';
+        $messageContent = 'testMessageContent';
+
+        $response = $this->createMock(ResponseInterface::class);
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/Pushy/Tests/PushyTransportTest.php]: Can you please use `JsonMockResponse`?

File: src/Symfony/Component/Notifier/Bridge/Pushy/composer.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=8.2",
+        "symfony/http-client": "^6.4|^7.0",
+        "symfony/notifier": "^6.4|^7.0"
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/Pushy/composer.json]:
```suggestion
        "symfony/notifier": "^7.1"
```

Comment (OskarStark) [reply src/Symfony/Component/Notifier/Bridge/Pushy/composer.json]: Otherwise low tests will fail

**Review by OskarStark — APPROVED** [review]
After my comments.

Can you please add a recipe and a docs PR? Thanks

File: src/Symfony/Component/Notifier/Bridge/Pushy/PushyOptions.php [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+use Symfony\Component\Notifier\Notification\Notification;
+
+/**
+ * @author Joseph Bielawski <stloyd@gmail.com>
+ *
+ * @see https://pushy.me/docs/api/send-notifications
+ */
+final class PushyOptions implements MessageOptionsInterface
+{
+    private const INTERRUPTION_LEVEL = ['passive', 'active', 'time-sensitive', 'critical'];
```
Comment (norkunas) [inline src/Symfony/Component/Notifier/Bridge/Pushy/PushyOptions.php]: Why not enum? :)

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @stloyd.

**Pull request merged**